default_prompts.py

```diff
@@ -40,9 +40,142 @@
     NO_CHECK_PROOF, NO_CHECK_CHEAT,
     ['Python'],
     None,
-    None
+    {
+        "def sum_unique_prime_factors(n):": "test(sum_unique_prime_factors(28) == 9)\ntest(sum_unique_prime_factors(45) == 8)\ntest(sum_unique_prime_factors(60) == 10)"
+    }
+)
+
+problem_python_2_oneshot = (
+    f"""### Spec: In {LANG}, write a 'magic(s)' function that: given a string, it finds all the characters that repeat an odd 
```

Comment:
Do you want to use quotes like this `magic`? Why say s explicitly?

Reply:
I removed the quotes from the magic. I think it is worthwhile to say magic(s) explicitly to force the LLM to generate the right name for the function. The unit tests depend on the right formatting of the function name.